### PR TITLE
Cleanup: make use of variable

### PR DESCRIPTION
**Pull request by greydnls**

### CODE REVIEW

File: src/Composer/Command/RequireCommand.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io = $this->getIO();
 
         $newlyCreated = !file_exists($file);
-        if (!file_exists($file) && !file_put_contents($file, "{\n}\n")) {
+        if (!$newlyCreated && !file_put_contents($file, "{\n}\n")) {
```
Comment (Seldaek) [inline src/Composer/Command/RequireCommand.php]: This one is a good example of a seemingly obvious fix that can bite one in the ass if one doesn't review carefully ;)

$newlyCreated is `!file_exists($file)` already so the `!` should be removed here if I'm not mistaken.

Comment (alcohol) [inline src/Composer/Command/RequireCommand.php]: Correct.

Comment (greydnls) [inline src/Composer/Command/RequireCommand.php]: Yeah, my bad there. Fixed it.

**Comment by Seldaek** [comment]
Heh no worries, thanks for the patch

**Pull request merged**